convert2rhel/systeminfo.py

```diff
@@ -152,7 +174,7 @@ def resolve_system_info(self):
     def print_system_information(self):
         """Print system related information."""
         self.logger.info("%-20s %s" % ("Name:", self.name))
-        self.logger.info("%-20s %d.%d" % ("OS version:", self.version.major, self.version.minor))
+        self.logger.info("%-20s %s" % ("OS version:", repr(self.version)))
```

Comment:
Should just be able to use `self.version` without repr
```suggestion
        self.logger.info("%-20s %s" % ("OS version:", self.version))
```

Reply:
You're right, thank you!